common/src/main/scala/uk/ac/wellcome/platform/archive/common/ingests/models/IngestUpdate.scala

```diff
@@ -5,21 +5,27 @@ import uk.ac.wellcome.platform.archive.common.bagit.models.BagVersion
 sealed trait IngestUpdate {
   val id: IngestID
   val events: Seq[IngestEvent]
+  val `type`: String
 }
 
-case class IngestEventUpdate(id: IngestID, events: Seq[IngestEvent])
-    extends IngestUpdate
+case class IngestEventUpdate(
+  id: IngestID,
+  events: Seq[IngestEvent],
+  `type`: String = "IngestEventUpdate"
```

Comment:
I’m 99% sure you don’t need to add this; Circe should do it for you automatically.

(See also: how we've been reading them off SQS queues all this time!)

Reply:
It's puzzling - the sns/sqs mechanism uses `IndividualMessageSender` which uses the `toJson` method from the JSON lib, that however does not seem to work if I use `toJson` to serialise in the tests.

Reply:
You are right though - this must be possible, i'd much rather not add the extra field to IngestUpdate.

I'll try again.

Reply:
Try importing `JsonUtil._`, I think that brings in the special code.